ui/components/multichain/pages/permissions-page/permissions-page.js

```diff
@@ -67,10 +61,9 @@ export const PermissionsPage = () => {
     );
   }, [totalConnections, sitesConnectionsList, snapsConnectionsList]);
 
-  const handleConnectionClick = useCallback((connection) => {
-    // TODO: go to connection details page
-    console.log('connection clicked: ', connection);
-  }, []);
+  const handleConnectionClick = (connection) => {
+    history.push(`/permissions/${connection.origin}`);
```

Comment:
I think these origins can looks like `https://app.uniswap.org`

If the slashes are problematic, we should encode the URL using `urlencode`